docs/api/assets/site.js

```diff
@@ -53,8 +53,8 @@ window.addEventListener('hashchange', function() {
 
```

Comment:
It looks like the directory containing this file is copied from `_theme/assets` when docs are built. Let's remove it from source control and gitignore the whole directory.